atomkraft/chain/node.py

```diff
@@ -72,11 +78,13 @@ def __init__(
             np.random.default_rng(list(bytes(final_seed))).bytes(strength // 8).hex()
         )
         self.coin_type = coin_type
+        coin = supported_coins.setdefault(self.coin_type)
+        if coin is None:
+            raise ValueError(
+                f"Provided coin type {self.coin_type} not supported."
+            )
 
-        class LocalCC(Cryptocurrency):
-            COIN_TYPE = CoinType({"INDEX": self.coin_type, "HARDENED": True})
-
-        self.wallet = hdwallet.BIP44HDWallet(cryptocurrency=LocalCC).from_entropy(
+        self.wallet = hdwallet.BIP44HDWallet(cryptocurrency=coin).from_entropy(
```

Comment:
I can't allow this change, as Cosmos community uses coin types others than 118.

Example,

| SLIP44 | Symbol | Chain         |
|--------|--------|---------------|
| 529    | SCRT   | Secret Network|
| 394    | CRO    | Crypto Chain  |
| 330    | LUNA   | Terra         |
| 852    | DSM    | Desmos        |

Reply:
maybe, add a logic, when coin is None, it falls back to `LocalCC`.